src/ApiPort/CommandLineOptions.cs

```diff
@@ -70,9 +70,17 @@ private static ICommandLineOptions ShowHelp(string suppliedCommand = null, bool
                 // TODO: Get invalid parameter (Microsoft.Framework.Configuration currently does not surface this)
                 Program.WriteColorLine($"Invalid parameter passed to {suppliedCommand}", ConsoleColor.Red);
             }
-
-            var codebase = new Uri(typeof(CommandLineOptions).GetTypeInfo().Assembly.CodeBase);
-            var path = Path.GetFileName(codebase.AbsolutePath);
+            
+            var location = typeof(CommandLineOptions).GetTypeInfo().Assembly.Location;
+            var path =
+#if NETCORE
```

Comment:
Actually, I just looked at this one and don't think that FEATURE_ASSEMBLY_LOCATION is right. I remember, now, why I used the general 'NETCORE' define instead of something more specific.

The difference here is whether or not we include the current file's extension in the example command lines in the help messages. I want to drop the extension in the CoreCLR case, since the extension will be .dll and that's not the file that users should be calling. In hosted cases, the user should actually call ApiPort or ApiPort.exe (depending on the platform), so I just leave the extension off.